views/policies.ecr

```diff
@@ -7,113 +7,120 @@
-                Valid units are Y(ear), M(onth), D(ays), h(ours), m(inutes), s(seconds).
-                Segments are only deleted when new messages are published to the stream queue.</span>
-            </a>
+        <section class="card regular-card">
+          <div class="table-wrapper">
+            <div id="table-error"></div>
+            <table id="table" class="table">
+              <thead>
+                <tr>
+                  <th>Virtual host</th>
+                  <th data-sort-key="name" class="left">Name</th>
+                  <th class="left">Pattern</th>
+                  <th class="left">Apply to</th>
+                  <th class="left">Definition</th>
+                  <th class="left">Priority</th>
+                  <th></th>
+                </tr>
+              </thead>
+              <tbody></tbody>
+            </table>
           </div>
-        </label>
-        <label>
-          <span>Priority</span>
-          <input type="number" name="priority" value="0" max="10" min="-10" step="1" required>
-        </label>
-        <button type="submit" class="btn-primary when-create">Add policy</button>
-        <div class="when-edit">
-          <button type="submit" class="btn-primary">Update</button>
-          <button type="reset" class="btn-danger cancel">Cancel</button>
-        </div>
-      </form>
-    </main>
-    <% render "partials/footer" %>
+        </section>
+        <form method="put" id="createPolicy" class="form card regular-card">
+          <h3 class="title-m">
+            <span class="when-create">Add policy</span>
+            <span class="when-edit">Edit policy</span>
+          </h3>
+          <div class="py-4">
+            <label class="when-create mb-4">
+              <span class="field-label">Virtual host</span>
+              <select class="dropdown" id="addPolicyVhost" name="vhost" required></select>
+            </label>
+            <label class="mb-4">
+              <span class="field-label">Name</span>
+              <input class="regular-text" type="text" name="name" required data-primary-key="">
+            </label>
+            <label class="mb-4">
+              <span class="field-label">Pattern</span>
+              <input class="regular-text" type="text" name="pattern" required>
+            </label>
+            <label class="mb-4">
+              <span class="field-label">Apply to</span>
+              <select class="dropdown" id="addPolicyApplyTo" name="apply-to" required>
+                <option value="all">Exchanges &amp; Queues</option>
+                <option value="exchanges">Exchanges</option>
+                <option value="queues">Queues</option>
+              </select>
+            </label>
+            <label class="mb-4">
+              <span class="field-label">Definition</span>
+              <textarea class="regular-textarea" name="definition" placeholder='{ "key": value }'></textarea>
```

Comment:
`class="regular-textarea"` why do we need to specify that it's regular? Isn't better to have a default styling for textareas, and if it's a "non-regular" textarea, then we add a class to it?